babel/messages/extract.py

```diff
@@ -111,10 +111,25 @@ def _strip(line: str):
     comments[:] = map(_strip, comments)
 
 
-def default_directory_filter(dirpath: str | os.PathLike[str]) -> bool:
```

Comment:
Would this function be considered part of the public API? If so, shouldn't we deprecate it first?